### PR TITLE
Add eurocrops entry

### PR DESCRIPTION
**Pull request by chorng**

### CODE REVIEW

**Comment by veseln** [comment]
I would change the description to: 

```
EuroCrops is a dataset collection combining all publicly available self-declared crop reporting datasets from countries of the European Union. 
The project is funded by the German Space Agency at DLR on behalf of the Federal Ministry for Economic Affairs and Climate Action (BMWK). 

This work is licensed under a [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).

An algorithm for retrieval of [Eurocrops](https://github.com/maja601/EuroCrops) Sentinel-2 signals based on an area of interest.

The algorithm allows the users to specify:
 * an area-of-interest (AoI);
 * a time interval for which they want to retrieve Sentinel-2 data;
 * an evalscript specifying which Sentinel-2 bands or derived indices, e.g. NDVI, to download.

Provided the above parameters, the algorithm computes Sentinel-2 time-series for the specified bands/indices for each polygon in the Eurocrops dataset that intersects the AoI. The computed signals are aggregated spatially over the polygon, and temporally over one day. 

Such signals constitute the basis for many applications, in particular in monitoring agricultural activity. Please refer to this [blog-post series](https://medium.com/sentinel-hub/area-monitoring-concept-effc2c262583) for examples on how these signals can be used.

The tool relies on AWS S3 bucket to store the results, thus read/write access to an AWS S3 bucket is required. 

The process consists of the following steps:

- Retrieve Eurocrops parcels for a chosen AoI. This is done with the help of [GeoDB](https://github.com/dcs4cop/xcube-geodb).
- The parcels are saved to the specified folder on the user's AWS S3 bucket. 
- Retrieve signals for the parcels in the AoI for the specified time interval using Sentinel Hub [Batch Statistical API](https://docs.sentinel-hub.com/api/latest/api/batch-statistical/).
- The signals are downloaded to the  user's AWS S3 bucket in the form of `.json` files.  

```

**Comment by veseln** [comment]
Looks okay.

File: collections/eurocrops.yaml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  - Croatia
+  - Lithuania
+  - Latvia
+  - Netherlands
+  - Portugal
+  - Romania
+  - Sweden
+  - Slovenia
+  - Slovakia
+TemporalAvailability:
```
Comment (Dorothyrono) [inline collections/eurocrops.yaml:45]: TemporalAvailabilty does not support table renderer

**Review by Dorothyrono — CHANGES_REQUESTED** [review]

File: collections/eurocrops.yaml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      - Name: AWS access key ID
+        Description: The AWS access key ID of the IAM user with read and write access to the bucket
+        Type: String
+      - Name: AWS secret access key
+        Description: The AWS secret access key of the IAM user with read and write access to the bucket
+        Type: String
+ProducedResults: |
+  The produced output are [Sentinel Hub Statistical API](https://docs.sentinel-hub.com/api/latest/api/statistical/) results for each parcel saved to the user's S3 bucket in the form of JSON files.
+Pricing: |
+  Calculation the price of an order is proportionate to the amount of data that will have to be processed. Processing Units (PUs) for this algorithm are directly proportional to:
```
Comment (Dorothyrono) [inline collections/eurocrops.yaml:121]: Small typo 

Calculation *of* the price....

File: collections/eurocrops.yaml [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  The tool relies on AWS S3 bucket to store the results, thus read/write access to an AWS S3 bucket is required. 
+  
+  The process consists of the following steps:
+    - Retrieve Eurocrops parcels for a chosen AoI. This is done with the help of [GeoDB](https://github.com/dcs4cop/xcube-geodb).
+    - The parcels are saved to the specified folder on the user's AWS S3 bucket. 
+    - Retrieve signals for the parcels in the AoI for the specified time interval using Sentinel Hub [Batch Statistical API](https://docs.sentinel-hub.com/api/latest/api/batch-statistical/).
+    - The signals are downloaded to the  user's AWS S3 bucket in the form of `.json` files.
+Image: eurocrops/thumbnail.png
+Explore: "[Run algorithm in EDC Browser](https://browser.eurodatacube.com/?zoom=10&lat=41.9&lng=12.5&fromTime=1970-01-01T00%3A00%3A00.000Z&toTime=2022-11-14T13%3A45%3A29.045Z&algorithm=eurocrops)"
+GeographicalCoverage:
```
Comment (Dorothyrono) [inline collections/eurocrops.yaml:28]: Add the pipe symbol `|`  to indent the list

```
GeographicalCoverage: |
  - Austria
  - Belgium
  - Germany
```

**Review by Dorothyrono — CHANGES_REQUESTED** [review]

File: collections/eurocrops.yaml (hunk 1 above)
Comment (chorng) [reply collections/eurocrops.yaml]: Fixed.

File: collections/eurocrops.yaml (hunk 2 above)
Comment (chorng) [reply collections/eurocrops.yaml]: Fixed.

File: collections/eurocrops.yaml [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+      - Name: Name
+        Title: Name
+      - Name: Description
+        Title: Description
+      - Name: Type
+        Title: Type
+    Rows:
+      - Name: aoi
+        Description: Area of interest
+        Type: Polyon or bounding box
```
Comment (Dorothyrono) [inline collections/eurocrops.yaml:76]: I missed this earlier sorry:) Just a small detail 

Adapt the attributes name to be exactly as it appears in EDC browser interface and also the order it appears : i.e 
- Area of Interest
- Time range
- Bucket Name
- Project folder
- AWS access key ID
- AWS secret access key
- Evalscript path

**Review by Dorothyrono — CHANGES_REQUESTED** [review]

File: collections/eurocrops.yaml (hunk 4 above)
Comment (chorng) [reply collections/eurocrops.yaml:76]: Fixed. Btw I checked [sice.yaml](https://github.com/eurodatacube/public-collections/blob/main/collections/sice.yaml) and [field-delineation.yaml](https://github.com/eurodatacube/public-collections/blob/main/collections/field-delineation.yaml) and it seems to me that the attribute name doesn't need to be the same as what is shown on EDC Browser.

Comment (Dorothyrono) [reply collections/eurocrops.yaml:76]: Thanks. everything works now. 

Merging

**Review by Dorothyrono — APPROVED** [review]

**Pull request merged**